Update Project Portfolio Page (#172)
CS2103-AY1819S1-W14-1/main: docs/DeveloperGuide.adoc

==== Current Implementation

> ClinicIO The login mechanism facilitated by using `UniqueStaffList` to retrieve a list of staffs. +

Reviewer: ```suggestion
ClinicIO's login mechanism is facilitated by `UniqueStaffList` to retrieve a list of staff. +
```

These operations are exposed in the Model interface as Model#hasStaff() and Model#checkStaffCred()  respectively.

Given below is a scenario to illustrate the login authentication at each step: +

Step 1: The user executes the `login` command by specifying the role, name and password. The `login` command determines the type of role entered by the user. +

> Step 2: The `login` command call `Model#hasStaff()` to ensure that this staff record exist inside ClinicIO. +

Reviewer: ```suggestion
Step 2: The `login` command calls `Model#hasStaff()` to ensure that this staff record exist inside ClinicIO. +
```
